src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -2441,6 +2457,11 @@ public Object exportCertificates(
             throw new IseException(
                 i18n.tr("Unable to create entitlement certificate archive"), e);
         }
+        catch (ConcurrentCertificateCreationException e) {
+            // TODO: Handle exception
```

Comment:
Is this also going to be handled in CHAINSAW-359?

Reply:
I didn't have the Jira at the time because it wasn't created yet, but it will be updated in CHAINSAW-376 and I updated the comment to add the Jira number